fms/utils/generation.py

```diff
@@ -149,6 +149,8 @@ def generate(
         use_cache: requires that the model accept use_cache and
             past_key_value_states args in forward method.
         eos_token_id: the optional token id representing the end of sequence
+        timing: whether to measure timings: "per-token" for each token generation time,
```

Comment:
Can you add the return types for time to the docstring so a user will know how to handle the output